set.mm

```diff
@@ -530418,31 +530444,6 @@ property of surreals and will be used (via surreal cuts) to prove many
     ( csur wcel cleft cfv csn csslt wbr cright c0 wne ssltleft ssltright sslttr
     snnzg syl3anc ) ABCADEZAFZGHRAIEZGHRJKQSGHALAMABOQRSNP $.
 
-  ${
-    $d A x y $.
-    $( The left options of a surreal are less than that surreal.  (Contributed
-       by Scott Fenton, 6-Aug-2024.) $)
-    llt $p |- ( A e. No -> ( _L ` A ) <<s { A } ) $=
-      ( vx vy csur wcel cleft cfv wss csn cv cslt wbr wral csslt leftf ffvelrni
-      cpw elpwid wa cvv snssi cbday cold crab leftval eleq2d breq1 elrab syl6bb
-      simplbda breq2 ralsng adantr mpbird ralrimiva w3a fvex snex pm3.2i brsslt
-      wb mpbiran syl3anbrc ) ADEZAFGZDHZAIZDHZBJZCJZKLZCVGMZBVEMZVEVGNLZVDVEDDD
-      QAFOPRADUAVDVLBVEVDVIVEEZSVLVIAKLZVDVOVIAUBGUCGZEZVPVDVOVIVJAKLZCVQUDZEVR
-      VPSVDVEVTVICAUEUFVSVPCVIVQVJVIAKUGUHUIUJVDVLVPVAVOVKVPCADVJAVIKUKULUMUNUO
-      VNVETEZVGTEZSVFVHVMUPWAWBAFUQAURUSBCVEVGUTVBVC $.
-
-    $( The right options of a surreal are greater than that surreal.
-       (Contributed by Scott Fenton, 6-Aug-2024.) $)
-    rgt $p |- ( A e. No -> { A } <<s ( _R ` A ) ) $=
-      ( vx vy csur wcel csn wss cright cfv cslt wbr wral csslt snssi cpw rightf
-      cv ffvelrni wa cvv elpwid cbday cold crab eleq2d breq2 elrab syl6bb simpr
-      rightval syl6bi ralrimiv wceq breq1 ralbidv ralsng mpbird w3a snex pm3.2i
-      fvex brsslt mpbiran syl3anbrc ) ADEZAFZDGZAHIZDGZBQZCQZJKZCVHLZBVFLZVFVHM
-      KZADNVEVHDDDOAHPRUAVEVNAVKJKZCVHLZVEVPCVHVEVKVHEZVKAUBIUCIZEZVPSZVPVEVRVK
-      AVJJKZBVSUDZEWAVEVHWCVKBAUJUEWBVPBVKVSVJVKAJUFUGUHVTVPUIUKULVMVQBADVJAUMV
-      LVPCVHVJAVKJUNUOUPUQVOVFTEZVHTEZSVGVIVNURWDWEAUSAHVAUTBCVFVHVBVCVD $.
-  $}
-
```

Comment:
Note: it looks like these were deleted since they were duplicates of ssltleft and ssltright